src/EditorFeatures/CSharpTest/EditAndContinue/BreakpointSpansTests.cs

```diff
@@ -1700,7 +1700,7 @@ void Goo()
     try
     {
     }
-    catch(Exception e) [|if$$ (e.Message != null)|]
+    catch(Exception e) [|when$$ (e.Message != null)|]
```

Comment:
"if (" as an exception clause was the original syntax.  These tests were not validating actual c# syntax cases.